contracts/DCAFeeManager/DCAFeeManager.sol

```diff
@@ -12,7 +12,50 @@ contract DCAFeeManager is Governable, IDCAFeeManager {
   /// @inheritdoc IDCAFeeManager
   IDCAHub public immutable hub;
 
-  constructor(IDCAHub _hub, address _governor) Governable(_governor) {
+  TargetTokenShare[] internal _distribution;
+
+  constructor(
+    IDCAHub _hub,
+    TargetTokenShare[] memory _distributionToSet,
+    address _governor
+  ) Governable(_governor) {
     hub = _hub;
+    setTargetTokensDistribution(_distributionToSet);
+  }
+
+  /// @inheritdoc IDCAFeeManager
+  function targetTokensDistribution() external view returns (TargetTokenShare[] memory) {
+    return _distribution;
+  }
```

Comment:
it didnt allow you to make the variable public? 😢

Reply:
I can make it public, but the getter will be by index, not the entire array. That's why I did it like this